AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/OutboxStatusEvent.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
```

Comment:
can you mention in comment for each Event payload, which event they are for? `/// Used as payload for the `outboxStatus` event, emitted to the Hub